riscv/src/compiler.rs

```diff
@@ -141,10 +137,6 @@ pub fn compile<T: FieldElement>(
     // Replace dynamic references to code labels
     replace_dynamic_label_references(&mut statements, &data_labels);
 
-    // Remove the riscv asm stub function, which is used
-    // for compilation, and will not be called.
-    statements = replace_coprocessor_stubs(statements, coprocessors).collect::<Vec<_>>();
```

Comment:
is this now done inside the runtime ?

Reply:
there is no more coprocessor stubs to replace: those `C` functions in the runtime became `ecall`s